docs/sources/flow/reference/components/module.string.md

```diff
@@ -70,11 +70,8 @@ source.
 
 ## Component health
 
-`module.string` is reported as healthy if the most recent load of the module was
-successful.
-
-If the module is not loaded successfully, the current health displays as
-unhealthy and the health includes the error from loading the module.
+`module.string` is reported as unhealthy if given an invalid configuration or if
```

Comment:
I feel like the new documentation here is hiding information. Nothing has changed, right? `module.string` is unhealthy if the most recent load of the module succeeded. 

Whether components can _start_ isn't part of the health definition, just if they can be created.